src/structure/ExtendedClient.ts

```diff
@@ -1,6 +1,6 @@
 import {Client, ClientOptions, Collection, Routes} from "discord.js";
 import {BaseCommand, ConfigType} from "./";
-import {PrismaClient} from "@prisma/client";
+import {PrismaClient} from "../../node_modules/.prisma/client";
```

Comment:
Why have you changed the path ?

Reply:
because i got this error `Module '"@prisma/client"' has no exported member 'PrismaClient'.ts(2305)`

Reply:
You had to do `npx prisma generate`

Reply:
yes i see